README.md

```diff
@@ -1 +1,51 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/AwesomeBooks-blue)
+
+# AwesomeBooks
+This project mainly outlines the implementation of JavaScript Class and modules which was in use to perform certain methods like Adding Books, Deleting Books, Showing list of Books and finaly performing all activities on Local Storage. It also uses Luxon library to show current time.
+
+# AwesomBook Screenshot 
+![](./screenshot2.png)
+
+## Built With
+
+- HTML
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## [LiveDemo](https://betses.github.io/Awesome-books-with-ES6/)
+
+### Prerequisites
+  - Install Node.js
+
+
+### Setup
+- clone the repository  - `git clone https://github.com/betses/Awesome-books.git`
+- `cd into the directory`
+- run Index.html in the browser
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@betses](https://github.com/betses)
+- Twitter: [@Betse_s](https://twitter.com/Betse_s)
+
+👤 **Author2**
+
+- GitHub: [@niicode](https://github.com/niicode)
+- Twitter: [@_niicode](https://twitter.com/_niicode)
```

Comment:
Great job @betses 🤝

- [ ] Kindly remove the second author from your readme since you contributed to this PR alone.